data/Music/Pokemon Series/Scarlet and Violet/Battle! vs Penny.meta

```diff
@@ -0,0 +1,4 @@
+Pokémon Scarlet/Violet - Battle! vs Penny

```

Comment:
This song is close to the line of being too loud. You might want to bring the volume down just a bit.